.github/workflows/integration-tests.yml

```diff
@@ -113,11 +116,65 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 14.x
+          node-version: 16.x
       - run: npx lerna bootstrap
       - run: npx lerna run compile
       - run: npx lerna run integration/local --stream
 
+  azure-deploy-integration-tests:
+    name: Azure Deployment Integration Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Az CLI login'
+        uses: azure/login@v1.4.0
+        with:
+          creds: '${{ secrets.AZURE_CREDENTIALS }}'
+
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

Comment:
I would use here 16 too, for consistency

Reply:
Good catch, I cherry-picked the changes and forgot to update this one. thanks